File: easy/1752.Check if Array is Sorted and Rotated.cpp
class Solution {
public:
    
    bool check(vector<int>& nums) {
        int n=nums.size();
        int ans=(nums[n-1]>nums[0]) ? 1 : 0 ;
        for(int i=0;i<n-1;i++) if(nums[i]>nums[i+1]) ans++;
        return ans<=1;
    }
};

File: easy/1752. Check if Array Is Sorted and Rotated.py
class Solution:
    def check(self, nums: List[int]) -> bool:
        n=len(nums)
        ans=1 if nums[-1]>nums[0] else 0
        for i in range(n-1):
            if nums[i]>nums[i+1]:
                ans+=1
        return ans<=1
